client/src/App.jsx

```diff
@@ -1,34 +1,15 @@
 import { React, useEffect, useState } from 'react';
-import { ReactDom, render } from 'react-dom';
+import RatingAndReview from './components/ratings-and-reviews/index';
 import Overview from './components/overview/Overview';
-import RelatedAndOutfit from './components/related-items-and-outfit/index.jsx';
+import RelatedAndOutfit from './components/related-items-and-outfit/index';
 
 function App() {
-  const [productId, setProductId] = useState(40346);
-
-  console.log('current id in app', productId);
+  const [productId, setProductId] = useState('');
 
   return (
-    <div>
-      <Overview />
-      <div>
-        Item-detail
-        {productId}
-      </div>
-      <p>...</p>
-      <p>...</p>
-      <div><RelatedAndOutfit productId={productId} setProductId={setProductId} /></div>
-      <p>...</p>
-      <p>...</p>
```

Comment:
Foresee a persistent merge problem here since all of us are editing this portion to preview our individual module in the browser. We should probably have a discussion about this portion.